collections/hub/golden_path/feature_tests/transfer_negative_scenarios/payee_invalid_timestamp.json

```diff
@@ -168,200 +169,94 @@
-                "exec": [
-                  "expect(callback.headers['accept']).to.equal(undefined)",
-                  ""
-                ]
-              },
-              {
-                "id": 16,
-                "description": "Callback Header - fspiop-uri is /quotes/quoteId",
-                "exec": [
-                  "expect(callback.headers['fspiop-uri']).to.equal('/quotes/' + request.body.quoteId)",
-                  ""
-                ]
-              },
-              {
-                "id": 17,
-                "description": "Callback Header - fspiop-http-method is PUT",
-                "exec": [
-                  "expect(callback.headers['fspiop-http-method']).to.equal('PUT')",
-                  ""
-                ]
-              },
-              {
-                "id": 18,
-                "description": "Callback Body - quoteId is null",
-                "exec": [
-                  "expect(callback.body.quoteId).to.equal(undefined)",
-                  ""
-                ]
-              },
-              {
-                "id": 19,
-                "description": "Callback Body - ilpPacket is not null",
-                "exec": [
-                  "expect(callback.body.ilpPacket).to.not.equal(undefined)",
-                  ""
-                ]
-              },
-              {
-                "id": 20,
-                "description": "Callback Body - condition is not nul",
-                "exec": [
-                  "expect(callback.body.condition).to.not.equal(undefined)",
-                  ""
-                ]
               }
             ]
           },
-          "params": {
-            "Type": "MSISDN",
-            "ID": "{$inputs.toIdValue}"
-          },
-          "path": "/quotes",
-          "url": "{$inputs.HOST_QUOTING_SERVICE}",
-          "scriptingEngine": "javascript",
+          "ignoreCallbacks": true,
           "scripts": {
+            "preRequest": {
+              "exec": [
+                ""
+              ]
+            },
             "postRequest": {
               "exec": [
-                "if(environment.ENABLE_JWS_VALIDATION) {",
-                "  requestVariables.jwsValidationStatus =  custom.jws.validateCallback(callback.headers, callback.body, environment.SIMPAYEE_JWS_PUB_KEY)",
-                "}",
-                "else if(environment.ENABLE_PROTECTED_HEADERS_VALIDATION) {",
-                "  requestVariables.protectedHeadersValidationStatus =  custom.jws.validateCallbackProtectedHeaders(callback.headers)",
-                "}",
-                "",
-                "if (environment.ENABLE_WS_ASSERTIONS===true) {",
-                "  requestVariables.payeeRequest = await websocket.getMessage('payeeRequest', environment.WS_ASSERTION_TIMEOUT)",
-                "}",
                 ""
               ]
+            }
+          }
+        },
+        {
+          "id": 4,
+          "description": "Send Quote - Response",
```

Comment:
How do we know its ok to send the Quote Response here?

I don't see any await inboundEvent listener in the `POST /Quotes` to determine it was received by the NORESPONSE_SIMPAYEE_NAME.

Otherwise we may have a race condition with the `PUT /Quotes` request below.

Reply:
Good point. Added listener to POST /quotes, here and throughout.